examples/draft-04/v2.0.0-draft.4/box.json

```diff
@@ -4,7 +4,7 @@
         "name": "Box Content API",
         "_postman_id": "48e1ee04-a604-2549-9422-43270e936399",
         "description": "I am trying to update this.",
-        "schema": "http://schema.getpostman.com/collection/v2/collection.json"
+        "schema": "http://schema.getpostman.com/json/collection/v2.0.0-draft.4/collection.json"
```

Comment:
Shouldn't this be like:
`https://schemas.getpostman.com/json/draft-<version>/collection/<version>/collection.json`?

Reply:
No. `draft4` links cannot be changed, they could break a lot of things.